wiki/howToUseSemantics.md

```diff
@@ -0,0 +1,76 @@
+<div id="main_content_wrap" class="outer">
+
+<section id="main_content" class="inner">
+
+## How to use semantics
+
+To import the project user need to add to pom.xml
+
+<div>
+
+    <repositories>
+       <repository>
+           <id>jitpack.io</id>
+           <url>https://jitpack.io</url>
+       </repository>
+    </repositories>
+    <dependency>
+        <groupId>com.github.Ericsson</groupId>
```

Comment:
Should it be capital letter E here?

Reply:
changed  com.github.Ericsson to com.github.eiffel-community